### PR TITLE
Design fixes/background color change

### PR DESCRIPTION
**Pull request by johnxnguyen**
**Changes:**
- renamed `ColorSchemeColorBackgroundNew` to `ColorSchemeColorConversationBackground`
- updated snapshot test references

The failing tests are the VoiceChannelOverlayTests, which were present before these changes

**Notes**:

The typing indicator background is now visible as it sits over both the conversation background and the input text view, is this ok? See the screenshot:

conversation background color typing indicator

### CODE REVIEW

**Comment by typfel** [comment]
The cell tests doesn't look correct. The gray color is only behind the text and not the whole cell.

**Comment by johnxnguyen** [comment]
I think because the `ConversationContentViewController` background isn't included in the snapshot test, which is the same colour as the `TextMessageCell` background

File: WireExtensionComponents/Utilities/ColorScheme.m [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 NSString * const ColorSchemeColorSeparator = @"separator";
 NSString * const ColorSchemeColorBackground = @"background";
-NSString * const ColorSchemeColorBackgroundNew = @"background-new";
+NSString * const ColorSchemeColorConversationBackground = @"background-new";
```
Comment (mikeger) [inline WireExtensionComponents/Utilities/ColorScheme.m]: I guess we can also rename the key, check `stylesheet.cas` line 1632 for where it needs to be renamed 👍

**Comment by mikeger** [comment]
🔥 Conflict to fix.

**Comment by johnxnguyen** [comment]
hey @daehn I've added the changes you requested

**Pull request merged**